lms/static/sass/course.scss.mako

```diff
@@ -41,10 +41,7 @@
 @import 'course/courseware/sidebar';
 @import 'course/courseware/amplifier';
 
-## Import styles for search
-% if env["FEATURES"].get("ENABLE_COURSEWARE_SEARCH"):
-    @import 'search/_search';
-% endif
+@import 'search/_search';
```

Comment:
@srpearce - there is no change to that currently. Bookmarks reuses search styling / architecture, but Search (as far as I know) isn't being enabled with this work.  

@muhammad-ammar - If bookmarks are to be enabled by default on the platform, it makes sense to remove the conditional statement as you have done. (Bookmark results rely on search styling, I now see why this conditional statement was removed.)